packages/data-bite/src/CdcDataBite.tsx

```diff
@@ -229,22 +310,20 @@ const CdcDataBite = (
         dataBite = Math.max(...numericalData);
         break;
       case DATA_FUNCTION_MIN:
-        dataBite = Math.min(...numericalData);
+        dataBite =Math.min(...numericalData);
         break;
       case DATA_FUNCTION_MODE:
-        dataBite = getMode(numericalData).join(', ');
+        dataBite = getMode(numericalData).join(''); // Needs to be properly tested !!!
```

Comment:
@Atash3000 - this looks like it's working. Thanks for calling it out you can safely remove the comment.